### PR TITLE
✨ feat(13.5.0): add support for CancelTradeOrders and CancelAutoGiroOrders methods

### PR DESCRIPTION
**Pull request by akobr**
It is about add support for:

- https://bricknode.atlassian.net/wiki/spaces/API/pages/3744595969/CancelAutoGiroOrders
- https://bricknode.atlassian.net/wiki/spaces/API/pages/1930133509/CancelTradeOrders

### CODE REVIEW

File: Bricknode.Soap.Sdk/Services/BfsOrderService.cs [hunk 1]
```diff
@@ -368,6 +387,54 @@ public async Task<CreateAutoGiroDepositOrderResponse> CreateAutoGiroDepositOrder
...
         }
 
+        /// <summary>
+        /// https://bricknode.atlassian.net/wiki/spaces/API/pages/3744595969/CancelAutoGiroOrders
+        /// </summary>
+        public async Task<CancelAutoGiroOrderResponse> CancelAutoGiroOrdersAsync(Guid[] orderIds, string? bfsApiClientName = null)
+        {
+            var request = await GetRequestAsync<CancelAutoGiroOrderRequest>(bfsApiClientName);
+
+            var listOfCancelTradeOrder = new List<CancelAutoGiroOrder>();
```
Comment (ATKelman) [inline Bricknode.Soap.Sdk/Services/BfsOrderService.cs:397]: This is autogiro but you named the variable "..TradeOrder"

File: Bricknode.Soap.Sdk/Services/BfsOrderService.cs [hunk 2]
```diff
@@ -368,6 +387,54 @@ public async Task<CreateAutoGiroDepositOrderResponse> CreateAutoGiroDepositOrder
...
+
+            foreach (var tradeOrderId in orderIds)
+            {
+                listOfCancelTradeOrder.Add(new CancelAutoGiroOrder
+                {
+                    OrderId = tradeOrderId
```
Comment (ATKelman) [inline Bricknode.Soap.Sdk/Services/BfsOrderService.cs:403]: This method does not map the cancelComment from the linked documentation, is that on purpose?

Comment (akobr) [reply Bricknode.Soap.Sdk/Services/BfsOrderService.cs:403]: I wanted to be consistent with CancelTradeOrders, where is one overload with just Ids and one with entire model.

File: Bricknode.Soap.Sdk/Services/BfsOrderService.cs (hunk 1 above)
Comment (akobr) [reply Bricknode.Soap.Sdk/Services/BfsOrderService.cs:397]: A copy&paste error, will fix it

Comment (akobr) [reply Bricknode.Soap.Sdk/Services/BfsOrderService.cs]: done

**Pull request merged**